packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -231,13 +232,14 @@ export interface IDropdownStyles {
   /** Refers to the callout that hosts Dropdown options in larger viewports. */
   callout: IStyle;
 
-  /** Subcomponent styles. */
+  /** SubComponent styles. */

```

Comment:
This should probably either be left as "Subcomponent" or changed to "Sub-component"

Reply:
👍 will get a fix. I actually was unsure about this change)))